### PR TITLE
Drawpile Icon

### PR DESCRIPTION
**Pull request by bach5000**
@fbrosson I'm doing this correctly, right? Make a new folder "additional-files" and add the icon into it?

Also, please ignore the bad quality icon. It's my first time using Icon-o-matic. I didn't know it was so difficult to use 😬

### CODE REVIEW

File: additional-files/Drawpile.rdef [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+resource(<your resource id here>) #'VICN' array {
```
[the remark this answers is not in the document]
Comment (bach5000) [reply additional-files/Drawpile.rdef]: @fbrosson: Oops :) I'll make a new PR with the files in the right directory.

[the remark this answers is not in the document]
Comment (fbrosson) [reply additional-files/Drawpile.rdef]: Okay. No problem.
(Remark: you could have updated your local branch and used `--force` to update your remote branch, and therefore avoid having to close this PR and open a new one, but that's OK. Don't worry.)

BTW, I just noticed that your PR was based on commits that you pushed to your master branch.
While this is not a problem at all, many users prefer to use [feature-branches](https://github.com/haikuports/haikuports/wiki/DevelopmentModel#what-are-feature-branches).

https://github.com/haikuports/haikuports/wiki/DevelopmentModel#what-are-feature-branches
This allows you to keep your master branch perfectly in sync with the project's master, and even easily refresh your master branch at any time, without having to touch your feature-branches.
That said, you are perfectly free to submit anything from your master branch. Using feature-branches is just a recommendation, and not using them does not have any impact.

[the remark this answers is not in the document]
Comment (bach5000) [reply additional-files/Drawpile.rdef]: @fbrosson Great! I'll keep that in mind 😄

**Comment by humdingerb** [comment]
When adding an icon via rdef, could also add an app signature, long and short description etc.
See e.g. any app's rdef in the Haiku source.

**Comment by Begasus** [comment]
Also bump the revision to the recipe to use the icon :)

**Comment by bach5000** [comment]
Please look at the new PR here: https://github.com/haikuports/haikuports/pull/3310
I did something with Git and something happened... so I kind of had to start over.

**Comment by Begasus** [comment]
This could be closed in favour of (duplicate) PR https://github.com/haikuports/haikuports/pull/3310?

**Pull request closed without merging**